en/sql/function/json_fn.rst

```diff
@@ -631,3 +631,120 @@ Note that json objects do not check containment the same way json arrays do. It
       json_contains('["a", "b1", ["a", {"k":["b2"]}]]','["b1", {"k":"b2"}]')
     ========================================================================
                                                                            1
+
+JSON_MERGE_PATCH
+===================================
+
+.. function:: JSON_MERGE_PATCH (json_doc, json_doc [, json_doc] ...)
+
+The **JSON_MERGE_PATCH** function merges 2 or more json docs and returns the resulting merged json. **JSON_MERGE_PATCH** differs from **JSON_MERGE_PRESERVE** in that it will take the second argument when encountering merging conflicts. **JSON_MERGE_PATCH** is compliant with
```

Comment:
```suggestion
The **JSON_MERGE_PATCH** function merges two or more json docs and returns the resulting merged json. **JSON_MERGE_PATCH** differs from **JSON_MERGE_PRESERVE** in that it will take the second argument when encountering merging conflicts. **JSON_MERGE_PATCH** is compliant with
```